GettingStarted.md

```diff
@@ -0,0 +1,181 @@
+          - `Add missing '@Override' annotations to implementations of interface methods`
+          - `Add missing '@Deprecated' annotations`
+          - `Remove unnecessary casts`
+    - Click the `Apply` button.
+  * Go to the `Java` -> `Editor` -> `Templates` page.
+    - Click the `Import...` button.
+    - Navigate to the `dev-config\codeStyle` subfolder of the **team-explorer-everywhere** repository.
+    - Select the `Java ALM Java Editor Templates.xml` file and click `Open`.
+    - Click the `Apply` button.
+
+1. Set Code Style preferences
+  * Go to the `Java` -> `Code Style` -> `Clean Up` page.
+    - Click the `Import...` button.
+    - Navigate to the `dev-config\codeStyle` subfolder of the **team-explorer-everywhere** repository.
+    - Select the `Java ALM Java Code Cleanup.xml` file and click `Open`.
+    - Make sure that `Java ALM` is selected in the `Active profile` list box.
+    - Click the `Apply` button.
+  * Go to the `Java` -> `Code Style` -> `Code Templates` page.
+    - Click the `Import...` button.
+    - Navigate to the `dev-config\codeStyle` subfolder of the **team-explorer-everywhere** repository.
+    - Select the `Java ALM Java Code Templates.xml` file and click `Open`.
+    - Click the `Apply` button.
+  * Go to the `Java` -> `Code Style` -> `Formatter` page.
+    - Click the `Import...` button.
+    - Navigate to the `dev-config\codeStyle` subfolder of the **team-explorer-everywhere** repository.
+    - Select the `Java ALM Java Code Formatting.xml` file and click `Open`.
+    - Make sure that `Java ALM` is selected in the `Active profile` list box.
+    - Click the `Apply` button.
+  * Go to the `Java` -> `Code Style` -> `Organize Imports` page.
+    - Click the `Import...` button.
+    - Navigate to the `dev-config\codeStyle` subfolder of the **team-explorer-everywhere** repository.
+    - Select the `Java ALM Java Code.importorder` file and click `Open`.
+    - Click the `Apply` button.
+
+### Import the projects into Eclipse
+
+1. Open Git Import Wizard using `File` -> `Import` -> `Git` ->`Projects from Git`.
+1. Select `Existing local repository`, click the `Add` button.
+  1. Using the `Browse...` button, navigate to your local copy of the **team-explorer-everywhere** repository, select it, and click the `Finish` button.
+1. Click the `Next` button to switch to the Wizard Selection page.
+1. Select the `Import existing Eclipse projects` option and the `source` folder under the root of the repository.
+1. Click the `Next` button.
+1. Make sure that all the `com.microsoft.tfs.*` projects are selected and click the `Finish` button.
+
+### Install Eclipse dependencies
+
+There are additional Eclipse-related dependencies that must be installed _for each Eclipse platform target version_.  For now, let's install the dependencies of earliest-supported Eclipse version (3.5.2).  If you ever need to target another version of Eclipse, repeat this process with the matching version.
```

Comment:
> 3.5 target supposed to be used with TFS on-premises only, but not with VSTS.

Commit 35348790e9c9db1e7dc41ce4fdfee3f8527be74f fixes that.  Again, I'd rather guide potential contributors into a path of success by pointing to the earliest-supported version of Eclipse, as opposed to a fix or new feature being submitted that only works against the latest version of Eclipse and we don't find out until it's too late.